### ISSUE TITLE
mermaidAPI: Can not parse classDiagram 

### ISSUE THREAD

**Issue by lisongmin** [knsv/mermaid#483]
I'd like integrate mermaid in my static site powered by hugo.
The mermaidAPI work well with graph and sequence diagram, but failed to class diagram.
thanks for help.

* error message:
```
Parse error on line 5:
...lass03 *-- Class04
---------------------^
Expecting 'CLASS', 'MEMBER', 'SEPARATOR', 'UNICODE_TEXT', 'NUM', 'ALPHA', got 'EOF'
```
* The code to reproduce:

```html
<html lang="zh-cn"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
      <link rel="stylesheet" href="https://cdn.rawgit.com/knsv/mermaid/7.0.0/dist/mermaid.forest.css">
  <script src="https://cdn.rawgit.com/knsv/mermaid/7.0.0/dist/mermaid.min.js"></script>
    <script>
        mermaidAPI.initialize({
            startOnLoad:false
        });
        $(function(){

            var elements = document.getElementsByClassName("language-mermaid");

            for (i in elements) {
                element = elements[i];
                var graphDefinition = element.innerText;
                var svg = mermaidAPI.render('ha_mermaid_' + i, graphDefinition, function(svg){});
                if (svg) {
                    element.innerHTML = svg;
                }
            }
        });
    </script>
</head>


<body>

<pre><code class="language-mermaid">graph TD
    A --> B

</code></pre>

<pre><code class="language-mermaid">classDiagram
    Class01 <|-- AveryLongClass
    Class03 *-- Class04
</code></pre>
</body>

```

**Comment by vozkych** [comment]
I noticed a similar behavior in my project. Except the error message. I have only a warning messages in the console. For every relation in the classDiagram I have the message:
```
17:07:18 (798) :WARN:  Adding relation: {"id1":"Class01","id2":"AveryLongClass","relation":{"type1":1,"type2":"none","lineType":0},"relationTitle1":"none","relationTitle2":"none","title":"Cool"}  mermaidAPI.min.js:14:5861
```

mermaidAPI.parse method returns 'false' for the classDiagram content, but doesn't throw an error to the mermaidAPI.parseError handler.

**Comment by llwt** [comment]
Running into this as well. Any luck on a workaround? We've had to switch back to PlanUML for class diagrams in our Gitbooks :(.

**Comment by bguan** [comment]
+1 really wanna see ClassDiagram working

**Comment by omainegra** [comment]
Still no progress on this?

**Comment by ruizmarc** [comment]
Same problem here with 7.1. I can render all graphs with mermaid.render except for ClassDiagram.

**Comment by jpetro416** [comment]
` Graph after layout (4) ["[*]", "Email_Node", "Next_Node", "fail"]
mermaid.js:99153  19:09:06.947 : WARN :  Node [*]: {"id":"[*]","label":"[*]","width":44.359375,"height":43,"x":22.1796875,"y":21.5}
mermaid.js:28888 Uncaught DOMException: Failed to execute 'querySelector' on 'Document': '#mermaid-1574640546947 #[*]' is not a valid selector.`

Seriously the more I use this, the more I don't understand why JSON wasn't used. This markdown stuff is not good and makes parsing dynamic stuff really annoying.

**Comment by knsv** [comment]
Hi I am sorry things are resisting your efforts... :)

Could you supply a generated example where the parsing fails in the liveeditor for me to look at?
https://mermaid-js.github.io/mermaid-live-editor/#/edit/
